tests/test_reweighting.py

```diff
@@ -225,6 +225,7 @@ def batched_u_fxn(samples, ligand_nb_params):
     return samples_0, samples_1, batched_u_0, batched_u_1, ref_params, ref_delta_f
 
 
+@pytest.mark.skip(reason="needs update since removal of lambda dependence in nonbonded potentials")
```

Comment:
TODO: investigate fix

Reply:
The skip annotations make sense for now, and I'll update the relevant functions in a follow-up!